torchdynamo/optimizations/backends.py

```diff
@@ -181,6 +181,21 @@ def fx2trt(model, inputs):
     return trt_mod
 
 
+@catch_errors
+def fx2trt_lower(model, inputs):
+    from torch.fx.experimental.fx2trt.lower import Lowerer
+    from torch.fx.experimental.fx2trt import LowerSetting
+
+    lower_setting = LowerSetting()
+    lower = Lowerer.create(lower_setting=lower_setting)
+    output = lower(model, inputs)
```

Comment:
`output` is never actually used (except to check return type of user code).  Do we need to update the return lines below?